api/operator/v1alpha1/certmanager_types.go

```diff
@@ -98,6 +102,22 @@ type CertManagerResourceRequirements struct {
 	Requests corev1.ResourceList `json:"requests,omitempty" protobuf:"bytes,2,rep,name=requests,casttype=ResourceList,castkey=ResourceName"`
 }
 
+// CertManagerScheduling describes the scheduling configurations for the cert-manager operands,
+// namely the controller, webhook and cainjector.
+type CertManagerScheduling struct {
+	// NodeSelector is a selector which must be true for the pod to fit on a node.
+	// Selector which must match a node's labels for the pod to be scheduled on that node.
+	// More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+	// +optional
+	// +mapType=atomic
+	NodeSelector map[string]string `json:"nodeSelector,omitempty" protobuf:"bytes,7,rep,name=nodeSelector"`
+
+	// If specified, the pod's tolerations.
```

Comment:
```suggestion
	// Tolerations are attached to the pods to tolerate any taint that matches
	// the triple <key,value,effect> using the matching operator <operator>.
	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
```

Reply:
This is from the official godoc: https://github.com/kubernetes/api/blob/v0.31.2/core/v1/types.go#L3875

Reply:
I think this should be updated as per your comment to provide more context for the users of CertManager CRD.

Reply:
The godoc now reads:
```go
	// Tolerations are attached to the pod to tolerate any taint that matches the
	// taint's key, value and effect using the toleration's matching operator.
	// More info: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
```